### ISSUE TITLE
Godog perform all test in subdirectory like go test ./...

### ISSUE THREAD

**Issue by mwywong** [DATA-DOG/godog#197]
I have got my features within a directory structures that can be multiple level deep. From the root of the tress when I run "go test ./..." it will run through all tests. 

Is there a similar way of calling godog that would traverse the directory structure like go test ./...

**Comment by l3pp4rd** [comment]
godog cannot run all subpackages at the moment. it is not an easy problem. go test has only one representation of the output and it runs every subpackage individually. godog would need to combine the output. plus all the profiling tools do not work for multi package testing. if for instance you use the progress formatter, you can write a Makefile or something to run all the packages based on one command. if godog would support such feature, it would first need to support all the output options it maintains and though, it would be harder for users to integrate a custom formatter and probably many other problems.

To summarize it - I do not think such feature can be implemented without making too much complexity. Instead, my suggestion is to write your own tool which would do the job by running godog in every package you have and dealing with output in your own preferred way.

**Comment by mwywong** [comment]
Thanks you for the explanation. Agree it become too complex for good use.
